web-registry/src/components/organisms/EcocreditsTable.tsx

```diff
@@ -10,18 +10,11 @@ import { formatDate } from 'web-components/lib/utils/format';
 
 import { truncate } from '../../lib/wallet';
 import { getAccountUrl } from '../../lib/block-explorer';
+import { GreyText } from '../atoms/GreyText';
+import { BreakText } from '../atoms/BreakText';
```

Comment:
while I love the idea of moving these to be reusable, it might be preferable to keep simple stuff like this within the context of the component it's used. 

For example, I might see these and want to combine the two, which could be done:
```tsx
<GreyText><BreakText>this text will be grey and break on word</BreakText></GreyText>
```
but the JSX would bloat quickly. We also have several forms of `grey` (grey.50, grey.100, info.main etc), and it's not clear which one this uses. 

However if there were something that was commonly used which required several styles, like... if we had some custom text that all tables use, it could make sense to have a `TableText` atom. 

Does that make sense? I love the idea, but worry it might get a bit confusing over time

Reply:
I personally see it as temporary. I think we have to do a good definition and refactoring of the Design System and the UI Primitives...

So I don't care, I don't have a preference. I remove the components and return it to its origin then?